src/main/java/ca/on/oicr/pineryreports/util/CommonOptions.java

```diff
@@ -0,0 +1,35 @@
+package ca.on.oicr.pineryreports.util;
+
+import org.apache.commons.cli.Option;
+
+public class CommonOptions {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)